api/src/test/scala/za/co/absa/loginsvc/rest/service/jwt/JWTServiceTest.scala

```diff
@@ -294,11 +294,11 @@ class JWTServiceTest extends AnyFlatSpec with BeforeAndAfterEach with Matchers {
     assert(initPublicKey._1 == jwtService.publicKeys._2.orNull)
   }
 
-  it should "phase out older keys after 8 seconds" in {
+  it should "phase out older keys after 30 seconds" in {
```

Comment:
These times are barely bearable, is it necessary to have such long-taking tests?